docs/notebooks/structure_function_showcase.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import matplotlib.pyplot as pylt
</pre><p>My understanding is that generally "plt" is the preferred shorthand here, not "pylt"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Updated!